ferry.go

```diff
@@ -564,7 +581,13 @@ func (f *Ferry) Run() {
 			s := <-c
 			if ctx.Err() == nil {
 				// Ghostferry is still running
-				f.ErrorHandler.Fatal("user_interrupt", fmt.Errorf("signal received: %v", s.String()))
+				if f.OverallState.Value() != StateCutover {
+					// Save state dump and exit if not during the cutover stage
+					f.ErrorHandler.Fatal("user_interrupt", fmt.Errorf("signal received: %v", s.String()))
+				} else {
+					// Log and ignore the signal during cutover
+					f.logger.Warn("Signal received: %s during cutover, will ignore and complete the move", s.String())
```

Comment:
```suggestion
					f.logger.Warnf("Signal received: %s during cutover, will ignore and complete the move", s.String())
```

To correctly format this

Reply:
Could you add an assertion for this log line in test?

Similar to https://github.com/Shopify/ghostferry/blob/5ec4ab591e3257f8f21bfa5b4d083746d688c66a/test/integration/trivial_test.rb#L33-L39

Reply:
Good catch, I'll fix it.